daemon/cmd/daemon.go

```diff
@@ -708,16 +708,25 @@ func newDaemon(ctx context.Context, cleaner *daemonCleanup, params *daemonParams
 		return nil, nil, fmt.Errorf("error while opening/creating BPF maps: %w", err)
 	}
 	// Upsert restored CIDRs after the new ipcache has been opened above
-	if len(restoredCIDRidentities) > 0 {
-		d.ipcache.UpsertGeneratedIdentities(restoredCIDRidentities, nil)
+	restoredResource := ipcacheTypes.NewResourceID(ipcacheTypes.ResourceKindDaemon, "", "")
+	if len(d.restoredCIDRs) > 0 {
+		// Asynchronously upsert the prefixes into the new ipcache map.
+		// By the time the datapath gets initialized to consume this
+		// map, ipcache should have completed the async work. If this
+		// doesn't occur, it will result in temporary drops in the
+		// time between datapath initialization until this async
+		// process completes.
+		// TODO: Add signalling mechanism to pass to the datapath to
```

Comment:
This should have a name associated with it? Is there a ticket?

Reply:
Yeah typically we would file a ticket. I'm exploring another option to solve the bug in the short term, so I'll leave this comment unresolved for now and revisit when picking this PR back up.